scripts/variantstore/wdl/CreateCohortTable.wdl

```diff
@@ -0,0 +1,91 @@
+version 1.0
+
+workflow CreateCohortTable {
+   input {
+        String project
+        String dataset
+
+        String? docker
+    }
+
+    # TODO update this docker source
+    String docker_final = select_first([docker, "us.gcr.io/broad-dsde-methods/variantstore:latest"])
+
+    call CreateCohortTableTask {
+        input:
+            project                         = project,
+            dataset                         = dataset,
+
+            docker                          = docker_final
+    }
+
+}
+
+task CreateCohortTableTask {
+    # indicates that this task should NOT be call cached
+    meta {
+       volatile: true
+    }
+
+    input {
+        String project
+        String dataset
+
+        String? query_project
+        String? destination_project
+        String? destination_dataset
+
+        String? destination_cohort_table_name
+        String? fq_cohort_sample_table
+        String? fq_sample_mapping_table
+
+        File? service_account_json
```

Comment:
is this still defined as a File even if it's just a path to a json in GCP?

Reply:
as Kris pointed out, _since_ it's a file, it'll localize, so we _can_ just pass it straight into python. tested!